dir.traversal.targets

```diff
@@ -99,5 +99,8 @@
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
 
   <Target Name="RestorePackages" DependsOnTargets="$(TraversalRestorePackagesDependsOn)" />
+  <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
```

Comment:
Was this moved from somewhere else? I thought these are already defined somewhere